rules/50-udev-default.rules

```diff
@@ -74,6 +74,8 @@ KERNEL=="tun", MODE="0666", OPTIONS+="static_node=net/tun"
 
 KERNEL=="fuse", MODE="0666", OPTIONS+="static_node=fuse"
 
+KERNEL=="kvm", GROUP="kvm", MODE="0666"
```

Comment:
Seems fine, but then I'd ask for dropping the uaccess rule from src/login/70-uaccess.rules instead, as it becomes redundant.